keanu-python/keanu/sequence.py

```diff
@@ -31,6 +31,18 @@ def add(self, vertex: Vertex, label: Optional[str] = None) -> None:
     def get(self, label: str) -> Vertex:
         return Vertex._from_java_vertex(self.unwrap().get(_VertexLabel(label).unwrap()))
 
+    def get_contents(self) -> Dict[str, Vertex]:
+
+        def get_unqualified_name_or_proxy_name(key, vertex) -> str:
+            if is_instance_of(k._gateway, vertex, "io.improbable.keanu.vertices.ProxyVertex"):
+                return "proxy_for." + key.getUnqualifiedName()
```

Comment:
Is this for when a proxy vertex is added to the sequence by the user? Can you add an example in the tests?

Reply:
So with sequences you have to add a proxy vertex which represents the value of a vertex in the last sequence item and then you redeclare (and add to the sequence item) the newer version of that vertex from this sequence. 
This means you will necessarily have two versions of an input/output vertex's label in a sequence item.

For instance see the last test in `test_sequence.py`. Does that make sense @icantsayrural